server/websocket.cpp

```diff
@@ -0,0 +1,213 @@
+#include <boost/beast/core.hpp>
+#include <boost/beast/websocket.hpp>
+#include <boost/asio/dispatch.hpp>
+#include <boost/asio/strand.hpp>
+#include <algorithm>
+#include <cstdlib>
+#include <functional>
+#include <iostream>
+#include <memory>
+#include <string>
+#include <thread>
+#include <vector>
+#include <map>
+
+#include <boost/beast/core/buffers_to_string.hpp>
+
+namespace beast = boost::beast;         // from <boost/beast.hpp>
+namespace http = beast::http;           // from <boost/beast/http.hpp>
+namespace websocket = beast::websocket; // from <boost/beast/websocket.hpp>
+namespace net = boost::asio;            // from <boost/asio.hpp>
+
+#include "websocket.hpp"
+#include "deadfish.hpp"
+
+class DfWebsocket;
+
+net::io_context ioc{1};
+tcp::acceptor acceptor_{ioc};
```

Comment:
I think that the name of this global variable shouldn't end with an underscore. This convention should be reserved for class members.